packages/template-retail-react-app/app/utils/jwt-utils.js

```diff
@@ -77,7 +79,7 @@ export async function jwksCaching(req, res, options) {
 
         // JWKS rotate every 30 days. For now, cache response for 14 days so that
         // fetches only need to happen twice a month
-        res.set('Cache-Control', 'public, max-age=1209600')
+        res.set('Cache-Control', 'public, max-age=1209600, stale-while-revalidate=86400')
```

Comment:
What was the reason we add this in?

Reply:
This allows us to use the cached response for 1 day while the server/CDN fetches a fresh response. I added this in to prevent errors from occurring when the cache expires and the server needs to call the `/jwks` endpoint again.